test/language/expressions/delete/identifier-strict-recursive.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2018 Mike Pennisi.  All rights reserved.
```

Comment:
That's a funny way of spelling your name ;)

Reply:
this is copied from the other file in this directory, all i did was change the delete operand. should i change the copyright?

Reply:
You created this file, so you should either add your name (such that both are there, on their own lines), or just change it to your name and this year.

Reply:
done